src/Stratis.Bitcoin.Features.Interop/InteropPoller.cs

```diff
@@ -474,8 +472,8 @@ private async Task PollCirrusForBurnsAsync(BigInteger blockHeight)
                                         RequestType = ConversionRequestType.Burn,
                                         Processed = false,
                                         RequestStatus = ConversionRequestStatus.Unprocessed,
-                                        Amount = new uint256(paddedArray),
-                                        BlockHeight = (int)blockHeight,
+                                        Amount = new uint256(transferRequestAmount.ToByteArray()),
```

Comment:
As mentioned before, this is not valid. You have to prepend a sufficient number of zeros for the uint256 constructor to accept the byte array. BigInteger basically only emits sufficient bytes to represent the internal integer, they aren't aligned to any particular size like uint256 is.

Because BigInteger is somewhat smarter about instantiation, you don't have to do the padding when converting from a uint256 to a BigInteger